File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-controls")
    print(f"Title of the page: {driver.title}")

    checkbox_element = driver.find_element(By.ID, "checkbox")

    if(checkbox_element.is_displayed()):
        print("Checkbox is visible before clicking toggle button as expected")
    else:
        print("Checkbox is not visible before clicking toggle button")

    driver.find_element(By.XPATH,"//button[text() = 'Toggle Checkbox']").click()

    if(checkbox_element.is_displayed()):
        print("Checkbox is visible after clicking toggle button")
    else:
        print("Checkbox is not visible after clicking toggle button as expected")

File: Selenium/Activities/Activity5.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity5 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		
		driver.get("https://training-support.net/webelements/dynamic-controls");
		System.out.println("Title of the page: " +driver.getTitle());
		
		WebElement checkboxElement = driver.findElement(By.id("checkbox"));
		
		if(checkboxElement.isDisplayed()) {
			System.out.println("Checkbox is visible before clicking toggle button as expected");
		}
		else {
			System.out.println("Checkbox is not visible before clicking toggle button");
		}
		
		driver.findElement(By.xpath("//button[text() = 'Toggle Checkbox']")).click();
		
		if(checkboxElement.isDisplayed()) {
			System.out.println("Checkbox is visible after clicking toggle button");
		}
		else {
			System.out.println("Checkbox is not visible after clicking toggle button as expected");
		}
		
		driver.quit();
	}
}
